### ISSUE TITLE
Unspent Outputs API. Outputs with 0 confirmation are not showed.

### ISSUE THREAD

**Issue by MeLoco1** [coinbase/toshi#177]
UTXO API doesn't show the outputs from non-confirmed transactions.
The same issue reported here https://github.com/coinbase/toshi/issues/148#issuecomment-69002611

**Comment by aalness** [comment]
Yes it would be nice if functionality were added to return unconfirmed outputs as well.
